### DISCUSSION TITLE
k = 1 or 2 for plot in multidimensional motif doc?

### DISCUSSION THREAD

**Discussion by hafarooki** [TDAmeritrade/stumpy#429] [Q&A] [answered]
https://stumpy.readthedocs.io/en/latest/Tutorial_Multidimensional_Motif_Discovery.html

In the hockey stick plot, it graphs the index of k and shows that the best is at 1. However, it says this means k=1 is the best. But at index 1 wouldn't it be k=2?

**Comment by seanlaw** [comment, marked as answer]
> In the hockey stick plot, it graphs the index of k and shows that the best is at 1. However, it says this means k=1 is the best. But at index 1 wouldn't it be k=2?

There may be a little bit of confusion here but `k` uses zero-based indexing and so `k = 1` corresponds to `2-dimensions` while, say, `k = 2` corresponds to `3-dimensions`
